src/lib/connectors/shared/remote-debugging-connector.ts

```diff
@@ -21,16 +21,16 @@ import { debug as d } from '../../utils/debug';
 import { delay, hasAttributeWithValue } from '../../utils/misc';
```

Comment:
@molant I've fixed `17:13  error  'r' is defined but never used  no-unused-vars` before merging.